example/src/components/RoomParticipant.tsx

```diff
@@ -24,40 +35,75 @@ export const RoomParticipant = ({
     return false;
   };
 
+  const getTextForPinButton = () => {
+    return focused ? 'Unpin user' : titleSmall ? 'Pin' : 'Pin user';
+  };
+
+  const onPinButton = () => {
+    if (focused) {
+      onPinButtonPressed(null);
+      return;
+    }
+    onPinButtonPressed(videoTrack!.id);
+  };
+
+  const triggerShowingPinButton = async () => {
+    if (pinButtonHiddden) {
+      return;
+    }
+
+    setShowPinButton(true);
```

Comment:
can we have this animated? a simple animation, just opacity transition

Reply:
Added FadeIn animation.

Reply:
You should also add fade out animation when button is disappearing

Reply:
I tried adding onExit={FadeOut} but that breaks other styles in the call, since the focused participant is shown in the middle of the animations, I can investigate more though .

Reply:
Fixed.